### DISCUSSION TITLE
Chose which disk is integrated in "media" folder of HA

### DISCUSSION THREAD

**Discussion by DW-homeassistant** [dianlight/hassio-addons#254] [Ideas]
Hi, 

Apparently, when the "medialibrary" is enable, all the additionnal disks are mounted in the Medias folder of HA. When using multiple disk, it would be useful to be able to chose which disk(s) has to be visible in medias.

For instance, I have one external disk for music/video and another for data... only the first one need to be connected in medias, and not the second one.

**Comment by dianlight** [comment]
Is already possible with the 'usage' parameter in acl section.

```
Option: usage (optional) (valid only for external disks)

Set the scope of the disk, usefull for ha network storage mount. Valid values are media,backup,share

Defaults to media for external disks if medialibray is enabled.
```

Probably I need to make a better documentation.

L

**Reply by AkoreJepade** [reply]
I remember seeing option for setting to "media, backup, share" but I no longer see those? Maybe they are in the WEB UI that no longer workers on my system?
